src/applications/accredited-representative-portal/accreditation/21a/pages/07-character-references-chapter/characterReferencesPages.js

```diff
@@ -36,12 +36,15 @@ const arrayBuilderOptions = {
     !item?.relationship,
   minItems: 3, // TODO: [Fix arrayBuilder minItems validation](https://app.zenhub.com/workspaces/accredited-representative-facing-team-65453a97a9cc36069a2ad1d6/issues/gh/department-of-veterans-affairs/va.gov-team/87155)
   maxItems: 4,
+  showSummaryDescription: true,
   text: {
     getItemName: item =>
       `${item?.fullName?.first} ${item?.fullName?.last}${
         item?.fullName?.suffix ? `, ${item?.fullName?.suffix}` : ''
       }`,
     cardDescription: item => `${item?.phone}, ${item?.email}`,
+    summaryDescription:
```

Comment:
I see that `summaryDescriptionWithoutItems` was introduced recently which looks to be serving the same purpose. Can you verify that we want to add a new prop vs reusing or repurposing the other one?

Reply:
New prop is desired, slightly different use cases (one for items present, one without)